File: src/c/fizzbuzz-6.c
// Author https://stackoverflow.com/users/1008428/martiert

#include <stdio.h>

int main(void)
{
    int i;
    for(i=1; i<=1000000; ++i)
    {
        if (i % 3 == 0)
            printf("Fizz");
        if (i % 5 == 0)
            printf("Buzz");
        if ((i % 3 != 0) && (i % 5 != 0))
            printf("number=%d", i);
        printf("\n");
    }

    return 0;
}

File: src/js/fizzbuzz-6.js
#!/usr/bin/nodejs

// Author https://gist.github.com/technowar
(function (){
    var array = [];

    for (var i = 1; i <= 100; i++) {
        array.push(i);

        if (i % 3 === 0) {
            array.splice(i - 1, 1, 'Fizz');
        }

        if (i % 5 === 0) {
            array.splice(i - 1, 1, 'Buzz');
        }

        if (i % 15 === 0) {
            array.splice(i - 1, 1, 'FizzBuzz');
        }
    }

    console.log(array);
})();
